versions/kruise-rollout/0.3/templates/rollouts.kruise.io_batchreleases.yaml

```diff
@@ -0,0 +1,265 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
https://helm.sh/docs/topics/charts/#custom-resource-definitions-crds 
Helm official doc suggest put all CRD yaml under folder `crds`

Reply:
> https://helm.sh/docs/topics/charts/#custom-resource-definitions-crds Helm official doc suggest put all CRD yaml under folder `crds`

Thank you for your suggestion. I have seen your relevant work, it's good! Bur it is expected that this PR will be merged soon. At that time, please rebase your codes, and I will not fix this issue here.  thanks again.